Content/ClosePrintSpooler.md

```diff
@@ -0,0 +1,22 @@
+# Close Print Spooler
+
+_ClosePrintSpooler_ is a sandbox setting that provided nuanced control over how sandboxed applications interact with the print spooler service.
+
+```ini
+[DefaultBox]
+ClosePrintSpooler=n
+```
+
+This setting can be used to prevent sandboxed applications from interacting with the print spooler service. When set to `y`, sandboxed applications will be unable to interact with the print spooler service - for example, print.
+
+Added as part of [0.5.4 / 5.46.0 version](https://github.com/sandboxie-plus/Sandboxie/blob/f3ab6a73a13a74f894b2d5ceb47ab69ee0581c97/CHANGELOG.md#054--5460---2021-01-06).
```

Comment:
```suggestion
Added as part of 0.5.4 / 5.46.0 version.
```

Reply:
Will that go on the "improve template" MR?

It's marginally better to have links IMHO :sweat_smile:

Reply:
In my opinion, putting direct links to the changelog, which in itself is huge, is a bit of a distraction.

Reply:
"You don't have to click it" :sweat_smile: - but sure